### PR TITLE
Dzelge xxx v0.7.0

### PR DESCRIPTION
**Pull request by dzelge**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Review by pont-us — CHANGES_REQUESTED** [review]
The environment.yml has a dependency `pyproj >=3.0` which isn't present in `meta.yaml`. Should this be added?

**Review by forman — CHANGES_REQUESTED** [review]
In `recipe/meta.yaml`, change `xcube_standard` into `xcube_plugins`

**Review by TonioF — APPROVED** [review]

**Comment by forman** [comment]
> The environment.yml has a dependency `pyproj >=3.0` which isn't present in `meta.yaml`. Should this be added?

Yes.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,20 +1,20 @@
...
+  sha256: d04f9cc1ffeef5929197b5f33ef7eceb031382370ba8091ff1cbc94393ec3816
 
 build:
   number: 0
   noarch: python
   script: {{ PYTHON }} -m pip install . -vv
   entry_points:
     - xcube = xcube.cli.main:main
-    - xcube_standard = xcube.plugin:init_plugin
+    - xcube_plugins = xcube.plugin:init_plugin
```
Comment (forman) [inline recipe/meta.yaml:17]: Remove this line entirely.

**Review by forman — CHANGES_REQUESTED** [review]

**Review by forman — APPROVED** [review]

**Review by pont-us — APPROVED** [review]

**Pull request merged**